src/components/CVPreview.tsx

```
@@ -1,171 +1,248 @@
+              <div className="flex justify-between items-start mb-2">
                 <div>
-                  <h3 className="text-xl font-semibold text-gray-900">{experience.companyName}</h3>
-                  <div className="mt-1 text-sm text-gray-600">
+                  <h4 className="text-gray-900 dark:text-white font-medium">{exp.companyName}</h4>
+                  <div className="text-sm text-gray-600 dark:text-gray-400">
                     {[
-                      experience.contractType && experience.contractType.replace('-', ' '),
-                      experience.workSchema && experience.workSchema.replace('-', ' '),
-                      experience.companySize,
-                      experience.companyIndustry
+                      exp.contractType && exp.contractType.replace('-', ' '),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)